web/src/PickNeighbourhoodMode.svelte

```diff
@@ -327,11 +309,18 @@
       >
         <Popup openOn="hover" let:props>
           <h2>{props.name}</h2>
-          <b>Area:</b>
-          {props.area_km2.toFixed(1)} km²
+
+          {#if selectedPrioritization == "none" || selectedPrioritization == "area"}
+            <b>Area:</b>
+            {props.area_km2.toFixed(1)} km²
+          {:else if selectedPrioritization == "simd"}
+            <b>Fake SIMD:</b>
```

Comment:
Nit: to really make sure nobody gets confused in the next few days, labeling this fake here too